content/blog/2025/04/2025-04-11-twim.md

```diff
@@ -0,0 +1,352 @@
+> [#4861]: https://github.com/matrix-org/matrix-rust-sdk/pull/4861
+> [#4800]: https://github.com/matrix-org/matrix-rust-sdk/pull/4800
+> [MSC4274]: https://github.com/matrix-org/matrix-spec-proposals/pull/4274
+> [MAS]: https://matrix.org/blog/2025/04/morg-now-running-mas/
+
+## Dept of Bots 🤖
+
+### maubot ([website](https://github.com/maubot/maubot))
+
+A plugin-based Matrix bot system.
+
+[HarHarLinks](https://matrix.to/#/@kim:sosnowkadub.de) says
+
+> This week I published a patch for my maubot plugin that resolves Spotify and other streaming services links to those of other platforms. This resolved a compatibility issue with the upstream API used. <https://github.com/HarHarLinks/maubot-idonthavespotify/releases/tag/v1.1.1>
+
+[wreck](https://matrix.to/#/@william:mssj.me) reports
+
+> more updates to my maubot [communitybot plugin](https://github.com/williamkray/maubot-communitybot), which aims to make it easy for community leaders to manage their rooms and members on matrix.
+> 
+> 1. big update to permission management: _cascading power-level management_! the bot will now pay attention to power-level changes made in your parent room or space, and replicate those changes to all rooms in the space that it can. yes, that means you can make someone a moderator of your space, and they will gain moderator permissions in all rooms in it! hooray! there's even a not-horribly-painful upgrade path to move your statically defined admins/mods in your config to be admins/mods of the space as long as your bot is a space admin.
+> 2. archive and replace rooms: have a room with members you can't kick out because of power level disfunction? enabled encryption on accident? suffering from whatever the latest state-resolution bug is? created a temporary room for a convention, gathering or date-specific happening, and don't remember how to send a custom tombstone event? easily archive rooms or create updated ones without ruining everyone's day!
+> 3. interactive verification workflow for new joiners: a new hip way to combat spam in your community rooms. you can now configure some or all of your rooms to require that new members verify they know how to accept a DM invite, and are capable of following simple instructions! migration command `!community verify-migrate` will fix your existing room's permissions safely (hopefully) so anyone already in your room doesn't need to verify.
+> 4. bulk message redaction: redact up to the last 100 messages from a user in a room on demand, or enable automatic bulk redaction in all your community rooms when a user is banned with the `redact_on_ban` setting. _finally_.
+> 
+> feel free to join my development channel and see first-hand how the interactive verification workflow works, ask questions (after you verify 😉), get help, or suggest features! [#dev:mssj.me](https://matrix.to/#/#dev:mssj.me)
+
+## Dept of Trust & Safety 🙆
+
+[Jim](https://matrix.to/#/@mackesque:matrix.org) announces
```

Comment:
Hi @jimmackenzie, thanks for your update! I appreciate the guidance about the hide images setting a lot since it's not always easy to find. When speaking neutrally on behalf of the Foundation, I would love if we could get closer to the ideal of including the info for clients beyond the element clients, perhaps starting with these we recommend on the website https://matrix.org/ecosystem/clients/. I appreciate if the T&S team has no capacity to research this at this moment. It would probably good to have an overview of T&S features in popular clients anyway both for the team and the public (https://github.com/matrix-org/matrix.org/issues/2685). If you or anyone would happen to have such info at hand and are able to share it (even in parts), we would be happy to add it on to this TWIM.

Reply:
I would **very much** like to include more inclusive guidance for all the clients in the ecosystem. That would be an excellent project for the T&S Working Group of the Governing Board (and for the folks who are building clients, I'd love to hear about your work on safety tooling).